core/tests/karma.conf.ts

```diff
@@ -106,9 +106,9 @@ module.exports = function(config) {
     autoWatch: true,
     browsers: ['CI_Chrome'],
     // Kill the browser if it does not capture in the given timeout [ms].
-    captureTimeout: 60000,
+    captureTimeout: 120000,
     browserNoActivityTimeout: 120000,
-    browserDisconnectTimeout: 60000,
+    browserDisconnectTimeout: 120000,
```

Comment:
Increased timeout because it was giving chrome headless error for original timeouts.